hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -223,6 +224,16 @@ export const PendingTables = () => {
     const isCancelOrderLoading = useSelector(cancelOrderLoadingSelector);
     const toggleVisibility = () => dispatch(togglePendingTable());
     const cancelOrder = (orderId: number) => dispatch(cancelOrderThunk(orderId));
+    const [showCancelOrderModal, setShowCancelOrderModal] = useState(false);
+
+    const closeModal = () => {
+        setShowCancelOrderModal(false);
+    };
+
+    const submitModal = (cancelOrderId: number) => {
+        cancelOrder(cancelOrderId); // Perform Cancellation
```

Comment:
I am assuming that this action is not ready for use currently right?

Reply:
This is ready, the comment is just there lol